native/src/components/CategoryList.tsx

```diff
@@ -54,8 +46,7 @@ const CategoryList = ({
   listContent,
   query,
   onItemPress,
-  language,
-  thumbnail
```

Comment:
If a prop is not used anymore at all it should also be removed from the prop type and not be passed to the component anymore.